content/en/docs/cluster-administration/platform-settings/notification-management/configure-slack.md

```diff
@@ -1,10 +1,85 @@
 ---
-title: "Configure Slack"
-keywords: 'KubeSphere, Kubernetes, custom, platform'
-description: ''
-linkTitle: "Configure Slack"
+title: "Configure Slack Notifications"
+keywords: 'KubeSphere, Kubernetes, Slack, notifications'
+description: 'Configure Slack notifications and add channels to receive notifications from alerting policies, kube-events, and kube-auditing.'
+linkTitle: "Configure Slack Notifications"
 weight: 8724
 ---
 
-TBD
+This tutorial demonstrates how to configure Slack notifications and add channels, which can receive notifications for alerting policies.
 
+## Prerequisites
+
+You have an available [Slack](https://slack.com/) workspace.
+
+## Obtain a Slack OAuth Token
+
+You need to create a Slack app first so that it can help you send notifications to Slack channels. To authenticate your app, you must create an OAuth token.
+
+1. Log in to Slack to [create an app](https://api.slack.com/apps).
+
+2. On the **Your Apps** page, click **Create New App**.
+
+3. In the dialog that appears, enter your app name and select a Slack workspace for it. Click **Create App** to continue.
+
+4. From the left navigation bar, select **OAuth & Permissions** under **Features**. On the **Auth & Permissions** page, scroll down to **Scopes** and click **Add an OAuth Scope** under **Bot Token Scopes** and **User Token Scopes** respectively. Select the **chart:write** permission for both scopes.
+
+   ![slack-scope](/images/docs/cluster-administration/platform-settings/notification-management/configure-slack/slack-scope.png)
+
+5. Scroll up to **OAuth Tokens & Redirect URLs** and click **Install to Workspace**. Grant the permission to access your workspace for the app and you can find created tokens under **OAuth Tokens for Your Team**.
+
+   ![oauth-token](/images/docs/cluster-administration/platform-settings/notification-management/configure-slack/oauth-token.png)
+
```

Comment:
After creating an app, you should invite it to the channel which will receive the notifications.